apps/src/lib/ui/ParentLetter.jsx

```diff
@@ -161,8 +191,19 @@ const SignInInstructions = ({loginType, sectionCode}) => {
     case SectionLoginType.picture:
       steps = (
         <ol>
-          <GoToSectionSignIn sectionCode={sectionCode} />
-          <li>Click on their picture password and then click 'Sign in'</li>
+          <GoToSectionSignIn
+            sectionCode={sectionCode}
+            studentName={studentName}
+          />
+          <li>
+            Click on their picture password and then click 'Sign in'
+            {secretPicturePath && (
+              <span>
+                <br />
+                <img src={secretPicturePath} style={{width: 60, margin: 10}} />
+              </span>
+            )}
+          </li>
           <li>
             If your student does not remember their picture password, please
             email me and I will provide it
```

Comment:
We can remove this line from the custom version/when we know the secretPicturePath since they have the info they need right above.

Reply:
Thank you!  This is a detail in the spec that I forgot to implement.